app/src/components/market/sections/market_create/steps/items/funding_and_fee_step.tsx

```diff
@@ -199,8 +200,26 @@ const FundingAndFeeStep: React.FC<Props> = (props: Props) => {
     submit,
     values,
   } = props
+
   const { arbitrator, category, collateral, funding, loadedQuestionId, outcomes, question, resolution, spread } = values
 
+  const { markets } = useGraphMarketsFromQuestion(loadedQuestionId || '')
+
+  const [currentToken, setCurrentToken] = useState({ exists: false, token: '', marketAddress: '' })
```

Comment:
Kinda superfluous, but it would be slightly more obvious if instead of `token`, we had `symbol`